libp2p/netMessenger.go

```diff
@@ -82,21 +81,17 @@ func init() {
 type networkMessenger struct {
 	p2pSigner
 	p2p.MessageHandler
+	p2p.ConnectionsHandler
```

Comment:
- move `Bootstrap()` method as well into connectionsHandler
- also move the `printLogsStats` rountine to connectionsHandler, including the `connectionsMetric`